cmd/kubeadm/app/util/marshal_test.go

```diff
@@ -135,7 +135,10 @@ func TestMarshalUnmarshalToYamlForCodecs(t *testing.T) {
 
 	kubeadmapiv1alpha3.SetDefaults_InitConfiguration(cfg)
 	scheme := runtime.NewScheme()
-	kubeadmapiv1alpha3.AddToScheme(scheme)
+	err := kubeadmapiv1alpha3.AddToScheme(scheme)
```

Comment:
```
if err := kubeadmapiv1alpha3.AddToScheme(scheme); err != nil {
...
}
```
`err != nil` can be moved on the same line.
marking this as [1]

Reply:
+1